src/components/toolbox/table/index.js

```diff
@@ -47,9 +47,9 @@ const Table = ({
         ))}
       </TableRow>
       )}
-      {data.map(row => (
+      {data.map((row, index) => (
         <TableRow
-          key={row.id}
+          key={index}
```

Comment:
💅  Index as a key is an anti-pattern
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md

Instead, you can do something like in delegates table which also doesn't have `id` coming from the API
https://github.com/LiskHQ/lisk-desktop/blob/c156288dbf5502e56bfe5153849ed1463147275d/src/components/shared/delegatesTable/index.js#L22

Or it might be cleaner to add another prop to the `Table` component that would specify what should be the `primaryKey`. used as `key`